src/routes/router.js

```diff
@@ -85,7 +88,7 @@ export default Router.extend({
   faq() { this.render(<FAQ/>) },
   subscribe(id) { this.render(<FirstSubscriptionPage planName={id}/>) },
   userShow(id) { this.render(<UserShow userId={id}/>, {backgroundColor: 'GREY'}) },
-  organizationShow(id) { this.render(<OrganizationShow organizationId={id} key={id}/>, {backgroundColor: 'GREY'}) },
+    organizationShow(id) { this.render(<OrganizationShow organizationId={id} key={id} tab={extractOrganizationTab(window.location.search)}/>, {backgroundColor: 'GREY'}) },
```

Comment:
This is just included for development, so hot reloading doesn't change tabs. Sorry for that not being clear, that was present in the original PR description. I was planning on removing this in prod as it is hacky.